test/inductor/test_torchinductor.py

```diff
@@ -2771,18 +2771,31 @@ def fn(a, b):
         if self.device != "cpu":
             self.assertEqual(torch._inductor.metrics.generated_kernel_count, 1)
 
-    def test_softmax_one_kernel(self):
+    @patch.object(config.triton, "persistent_reductions", True)
```

Comment:
Might be worth adding one of the "triton-code based" tests that @eellison added some infra for: https://github.com/pytorch/pytorch/pull/93810/files#diff-809c39aeafb3acc92289f42a63e670a8719d4ce5627d5f88820142d80edf8d2aR6610